openedx/frontend-app-authoring
# refactor!: turn on homepage course API V2 consumption by default

## Description

Turn on getting courses from the HomePageCourses API which allows pagination, filtering and ordering. See https://github.com/openedx/edx-platform/pull/34173 for more details on the API implementation.

When we implemented https://github.com/openedx/frontend-app-authoring/pull/846, we [agreed to turn this feature back on](https://github.com/openedx/frontend-app-authoring/pull/846#issuecomment-2040277279) once Redwood was cut, so this change is long overdue. 

## Supporting information

Please, see https://openedx.atlassian.net/wiki/spaces/OEPM/pages/4032856101/Studio+Home+incremental+upgrades+-+product+approach

## Testing instructions

This view should be on by default after this change:
![image](https://github.com/user-attachments/assets/107e50aa-7d62-4f0c-8e1f-111f6f17c51a)
